helerm/urls.py

```diff
@@ -45,7 +46,21 @@
 router.register(r"record-search", RecordSearchDocumentViewSet, basename="record_search")
 router.register(r"all-search", AllSearchDocumentViewSet, basename="all_search")
 
+
+#
+# Kubernetes liveness & readiness probes
+#
+def healthz(*_, **__):
+    return HttpResponse(status=200)
+
+
+def readiness(*_, **__):
```

Comment:
Would it be beneficial to have `@never_cache` for these?

Reply:
Good idea.